iothub/device/src/Transport/Http/ServicePointHelpers.cs

```diff
@@ -41,8 +41,11 @@ public static void SetLimits(HttpMessageHandler messageHandler, Uri baseUri, int
                     ServicePoint servicePoint = ServicePointManager.FindServicePoint(baseUri);
                     servicePoint.ConnectionLeaseTimeout = connectionLeaseTimeoutMilliseconds;
                     break;
-#if NETCOREAPP2_1_OR_GREATER || NET5_0_OR_GREATER
-                // SocketsHttpHandler is only available in netcore2.1 and onwards
+#if NET5_0_OR_GREATER
```

Comment:
So this includes .net core 2.1?

Reply:
It does not since we do not target .net core 2.1

Reply:
I think there's a case to be made for the preprocessing directive to be true to the code and free of what our actual targets are. If the targets were to change, and I admit it is unlikely, then one doesn't need to recheck all the preprocessing directives for assumptive logic.

Reply:
Fair point. I can add .NET core 2.1 back in here.